.travis.yml

```diff
@@ -36,9 +44,23 @@ before_script:
   - sh -c 'if [ "$SYMFONY_VERSION" != "" ]; then composer require -n --no-update symfony/symfony=$SYMFONY_VERSION; fi;'
   - composer install --prefer-source
 
-  - sudo apt-get update
-  - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5
-  - sudo sed -i -e "s,/var/www,$(pwd)/Tests/Resources/public,g" /etc/apache2/sites-available/default
-  - sudo /etc/init.d/apache2 restart
+  - sudo chmod -R 0777 Tests/logs
+  - sudo apt-get update > /dev/null
+  - sudo apt-get install -y --force-yes apache2 libapache2-mod-fastcgi > /dev/null
+  # enable php-fpm
+  - sudo cp ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf.default ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf
+  - sudo a2enmod rewrite actions fastcgi alias
+  - echo "cgi.fix_pathinfo = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - ~/.phpenv/versions/$(phpenv version-name)/sbin/php-fpm
+  # configure apache virtual hosts
+  - sudo cp -f Tests/Resources/travis-ci-apache /etc/apache2/sites-available/default
+  - sudo sed -e "s?%TRAVIS_BUILD_DIR%?$(pwd)/Tests?g" --in-place /etc/apache2/sites-available/default
+  - sudo /etc/init.d/apache2 restart || (sudo cat /var/log/apache2/error.log && exit 1)
```

Comment:
you should never use ``exit`` in a ``.travis.yml``. All commands are used to generate a single build script, so using exit would exit the whole build script, breaking the Travis setup (errors won't be displayed properly, etc...). Use ``return 1`` instead